src/app/components/FlowchartGenerator.tsx

```
@@ -195,7 +221,7 @@ function FlowchartGenerator({
     setRenderError(null);
   }, [initialCode]);
   
-  // Use reaflow if available, otherwise fall back to mermaid
+  // Use reaflow if available and valid, otherwise fall back to mermaid
   const hasValidFlowchart = mermaidCode && mermaidCode.includes('-->');
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)